appsdk/sdk.go

```diff
@@ -393,9 +391,24 @@ func (sdk *AppFunctionsSDK) Initialize() error {
 	sdk.webserver = webserver.NewWebServer(sdk.config, sdk.secretProvider, sdk.LoggingClient, mux.NewRouter())
 	sdk.webserver.ConfigureStandardRoutes()
 
+	sdk.LogStartUpMessage(dic, startupTimer)
+
 	return nil
 }
 
+// LogStartUpMessage logs the service startup message
+func (sdk *AppFunctionsSDK) LogStartUpMessage(dic *di.Container, startupTimer startup.Timer) {
+	sdk.LoggingClient.Info("Service dependencies resolved...")
+	sdk.LoggingClient.Info(fmt.Sprintf("Starting %s %s ", sdk.ServiceKey, internal.ApplicationVersion))
```

Comment:
This one not useful here as `Starting...` since it come right before the `Started message`. Lets move this message to the beginning of Initialized() right after the call to setServiceKey().